Eastern-Research-Group/expert-query
# Bugfix/258 state filter issue

## Related Issues:
* [EQ-258](https://jira.epa.gov/browse/EQ-258)

## Main Changes:
* Fixed the issue of the `state` column options only filtering by value when populated asynchronously. This was accomplished by creating a `states` utility table that is used to add a `statename` column to the materialized view for the corresponding values query.
* Updated the `state` option labels to show the state code and the state name, since both can be used to search.

## Steps To Test:
1. Run the full ETL while pointed at a local database.
2. In the UI, select a profile and search for a state. Verify that it is possible to search by state name and state code.
3. Select a region, and repeat step 2.
